includes/class-wcsr-resource-manager.php

```diff
@@ -110,27 +110,20 @@ public static function deactivate_resource( $external_id ) {
 	 */
 	public static function maybe_prorate_renewal( $renewal_order, $subscription ) {
 
+		$is_prorated  = false;
 		$resource_ids = WCSR_Data_Store::store()->get_resource_ids_for_subscription( $subscription->get_id() );
 
 		if ( ! empty( $resource_ids ) && count( $resource_ids ) > 1 ) {
```

Comment:
Should this be changed to also calculate when there is a single resource on a subscription not just `>1`. Sorry if i'm missing something obvious.

Reply:
Ah yes definitely.

How we use it for RN  will depend on what we decide about how to handle the pre-payed amount (i.e. if that is accounted for using a product line item or a fee line item). But I think it should be removed here, and if we decide to use a different line item, we can handle that explicitly in the `'wcsr_after_renewal_order_prorated'` hook.

Reply:
@jconroy I've fixed that up with SHA: 43f6c63